### PR TITLE
passed tests for ch08

### PR DESCRIPTION
**Pull request by alsosteve**

### CODE REVIEW

File: python/tests/code_challenges/test_linked_list_zip.py [hunk 1]
```diff
@@ -85,4 +80,4 @@ def test_b_empty():
     expected = LinkedList()
     for value in reversed([1, 2, 3]):
         expected.insert(value)
-    assert actual == expected
+    assert str(actual) == str(expected)
```
Comment (aaron-imbrock) [inline python/tests/code_challenges/test_linked_list_zip.py:83]:
```
 - [ ] Top-level README ?Table of Contents? is updated
 - [ ] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
       - [ ] API and Unit Tests discussed
 - [ ] Feature tasks for this challenge are completed
       - [ ] Write a function called zip lists
       - [ ] rguments: 2 linked lists
       - [ ] Return: New Linked List, zipped as noted below
       - [ ] Zip the two linked lists together into one so that the nodes alternate between the two lists and return a reference to the the zipped list.
       - [ ] Try and keep additional space down to O(1)
       - [ ] You have access to the Node class and all the properties on the Linked List class as well as the methods created in previous challenges.
 - [ ] Unit tests written and passing 
       - [ ] ?Happy Path? - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)
 - [ ] Submit Requirements
       - [ ] Copy the link to this open pull request and paste it into the assignment submission field.
       - [ ] Left a description of how long this assignment took in the comments box.
       - [ ] Left additional comments to your grader about the process or any difficulties had with the assignment (if relevant).


```
